MOBILE_API.xml

```diff
@@ -3026,6 +3026,7 @@
         <element name="APP_SERVICES" since="5.1"/>
         <element name="SEAT_LOCATION" since="6.0"/>
         <element name="DISPLAYS" since="6.0"/>
+        <element name="DRIVER_DISTRACTION" since="7.0"/>
```

Comment:
I believe this needs to be added to `HMICapabilities` as well, these values have been desynced in the past. This would likely require a proposal revision though